pkg/mock/mock_pgxconn_ifase.go

```diff
@@ -0,0 +1,80 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: ../Documents/spqr/pkg/datatransfers/temp.go
```

Comment:
where is `datatransfers/temp.go` file itself? It should not be dropped. 
Also, we need Makefile target which runs `go generate .... ` on these files